docs/resources/scenarios/punch-clock.md

```diff
@@ -0,0 +1,142 @@
+---
+title: 'Office Scripts sample scenario: Punch clock button'
+description: A sample that lets a user press a button to record the current time.
+ms.date: 04/26/2022
+ms.localizationpriority: medium
+---
```

Comment:
Could we give a shoutout somewhere here to Brian who contributed this script? I liked what we did here with Leslie's sample script: https://docs.microsoft.com/en-us/office/dev/scripts/resources/samples/community-seasons-greetings - perhaps we could link to Brian's script repo here: https://github.com/b-gonzalez/Office-Scripts/